package.json

```diff
@@ -11,11 +11,13 @@
     "build-prod-readable": "yarn run build:clean && NODE_ENV=production webpack --progress",
     "build-docs": "documentation build src/types/profile.js -f md > docs/processed-profile-format.md; cat docs/processed-profile-format.md",
     "eslint": "eslint index.js src",
```

Comment:
I definitely prefer "lint", that's what I'm used to as well :) But because I don't really mind I didn't want to change existing naming ;) But 👍 if you'd like this change